scripts/backup_outputs.py

```
@@ -0,0 +1,41 @@
+
+current_dir = Path(os.getcwd())
+
+if os.path.basename(current_dir.resolve()) != "clp-bench":
+    raise Exception("Must be run in clp-bench directory, if it was renamed, comment this line out")
+
+def backup_outputs():
+    assets_dir = "assets"
+    output_base = "outputs"
+
+    # Generate timestamp only once per run
+    timestamp_str = datetime.now().strftime("%Y%m%d_%H%M%S")
```

Ruff (0.12.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)